src/v2/controllers/vdeployment_controller/trusted/spec_types.rs

```diff
@@ -127,9 +125,11 @@ impl ResourceView for VDeploymentView {
         // template and its metadata ane spec exists
         &&& self.spec.template.is_Some()
         &&& self.spec.template.get_Some_0().metadata.is_Some()
+        // can be derived from selector match labels
+        &&& self.spec.template.get_Some_0().metadata.get_Some_0().labels.is_Some()
```

Comment:
I think deployment's template should not be option: https://github.com/Arnavion/k8s-openapi/blob/master/src/v1_30/api/apps/v1/deployment_spec.rs#L28